### PR TITLE
Edit descriptions/names of 5 addons

### PR DESCRIPTION
**Pull request by DNin01**
### Changes

Check `Files changed` for the changes.

### Reason for changes

These changes are meant to make a few of the addons easier to find, understand, and use.

### Tests

Should work.

### CODE REVIEW

File: addons/hide-new-variables/addon.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Hide new variables",
-  "description": "Do not automatically create monitors for newly created variables or lists.",
+  "description": "Do not automatically show new variables or lists on the project player.",
```
Comment (mxmou) [inline addons/hide-new-variables/addon.json:3]:
```suggestion
  "description": "Do not automatically show new variables or lists on the stage.",
```

File: addons/pause/addon.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Pause button",
+  "name": "Project pause button",
```
Comment (mxmou) [inline addons/pause/addon.json:2]: I think short names are better for simple addons like this. You can add more detail to the description if you think it's necessary, for example "Adds a button next to the green flag that pauses the project."

File: addons/pick-colors-from-stage/addon.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Select stage colors in the costume editor",
-  "description": "Allows the costume editor's eye dropper to also pick colors from the stage.",
+  "name": "Pick colors on stage with eyedropper",
+  "description": "Allows the costume editor's eyedropper tool to also pick colors from the stage.",
```
Comment (mxmou) [inline addons/pick-colors-from-stage/addon.json:3]: "Tool" usually means a mode like select or brush.

File: addons/pause/addon.json (hunk 2 above)
Comment (DNin01) [reply addons/pause/addon.json:2]: Other related addons such as "muted project player mode" and "project volume slider" have "project" in their names.

Comment (WorldLanguages) [reply addons/pause/addon.json:2]: It's obvious that "pause button" refers to projects. What else in the Scratch website could be paused?

**Review by mxmou — APPROVED** [review]

**Review by Samq64 — APPROVED** [review]

File: addons/hide-new-variables/addon.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Hide new variables",
-  "description": "Do not automatically create monitors for newly created variables or lists.",
+  "description": "Do not automatically show new variables or lists on the stage.",
```
Comment (WorldLanguages) [inline addons/hide-new-variables/addon.json:3]: So are we going to avoid using the word "monitor"?

Comment (Samq64) [reply addons/hide-new-variables/addon.json:3]: I don't think we need to, but I think this description is fine.

Comment (DNin01) [reply addons/hide-new-variables/addon.json:3]: Here are a few variations that use the word “monitors”:
```suggestion
  "description": "Do not automatically show variable and list monitors on the stage.",
```
```suggestion
  "description": "Do not automatically show monitors for new variables or lists.",
```
```suggestion
  "description": "Do not automatically show monitors for newly created variables or lists.",
```

Or one other option:
```suggestion
  "description": "Hide newly created variables and lists by default.",
```

Comment (BroJac5246) [reply addons/hide-new-variables/addon.json:3]: I like the third one best, as it's clear and uses 'monitor.'

Comment (WorldLanguages) [reply addons/hide-new-variables/addon.json:3]: I didn't mean to say we should be using the word "monitor". Honestly, I think we should not

**Comment by WorldLanguages** [comment]
Is there enough consensus here?

**Comment by WorldLanguages** [comment]
Regarding the use of the word "monitor", a new addon was added after the creation of this PR, which uses it for the description (copy-reporter).
Are we actively trying to avoid using that word? If so, we should also edit that description on this PR (git pull master first)

**Comment by DNin01** [comment]
> Regarding the use of the word "monitor", a new addon was added after the creation of this PR, which uses it for the description (copy-reporter). Are we actively trying to avoid using that word? If so, we should also edit that description on this PR (git pull master first)

I think there are cases, including `copy-reporter`'s description where using the word "monitor" makes more sense than not.

**Comment by mxmou** [comment]
The word "monitor" was used in the "show variable" block's help text in 2.0.
image

I don't think it's used anywhere in Scratch 3.0's UI, but a lot of Scratchers use it and might want to be able to search for it on the settings page. There are currently two addons that mention monitors in the description: "hide new variables" and "copy reporter values".

**Comment by WorldLanguages** [comment]
> There are currently two addons that mention monitors in the description: "hide new variables" and "copy reporter values".

Note that "hide new variables" would stop mentioning monitors if we merged this PR in the current state (unless I'm missing something). So searching for "monitor" would only match 1 addon.

**Comment by DNin01** [comment]
> So searching for "monitor" would only match 1 addon.

That's something to think about.

**Comment by WorldLanguages** [comment]
I think we could add (keep) the word "monitor" in hide-new-variable's description before merging this

**Review by WorldLanguages — APPROVED** [review]

**Pull request merged**